baggers.js

```diff
@@ -14194,6 +14194,102 @@ var data = {
+          "tags": [ "Agile", "Amazon", "Architecture", "Automatisation", "AWS", "Azure", "Cloud", "Continuous Delivery", "DevOps", "Docker", "Git", "Ingénierie agile", "IT automation", "Monitoring", "Ops", "Outils", "Performance", "Scalabilité", "Sécurité", "Stratégie", "supervision", "Usine logicielle" ],
+          "lang": [ "fr", "en", "es" ]
+        },
+        {
+          "title": "Monitoring et Alerting opensource",
+          "abstract": "Si vous pensez qu'une solution de monitoring est forcement payante et/ou en mode SaaS ? Je vous propose de vous prouver le contraire ou comment via des outils opensource simple, puissants et ouverts, vous pouvez adresser vos problématiques de qualité de service.",
+          "tags": [ "Architecture", "Asynchrone", "Automatisation", "Cloud", "Elasticsearch", "Ingénierie agile", "IT automation", "Kibana", "KPI", "Métrique", "Monitoring", "Open Source", "Outils", "Performance", "Scalabilité", "supervision" ],
+          "lang": [ "fr", "en", "es" ]
+        },
+        {
+          "title": "Opensource pour les administrations et les collectivités locales",
+          "abstract": "En France, au Royaume Uni, aux Etats Unis et partout dans l’Union Européenne, les gouvernements ont formulé des recommandations favorables à l’usage de logiciels open source dans leurs administrations. Le choix de systèmes d'information et l'interoperabilité de ceux-ci, sont la trâme future de notre bien commun numérique national. Maintenant comment on fait ;-) ?",
+          "tags": [ "Architecture", "Asynchrone", "Automatisation", "Build", "Cloud", "Continuous Delivery", "Cryptographie", "DevOps", "Git", "Industrialisation", "Ingénierie agile", "Intégration", "IT automation", "Micro-Services", "MicroServices", "Outils", "Sécurité", "Stratégie", "Usine logicielle" ],
+          "lang": [ "fr", "en", "es" ]
+        },
+        {
+          "title": "Offshore programming",
+          "abstract": "20 années d'expérience en Offshore programming (Biélorussie, Ukraine, Egypte, Chine, Inde, Maroc, Colombie, Sri Lanka....). Le pays et la langue importent beaucoup moins que la démarche, les outils de suivi, et l'adéquation entre votre niveau d'industrialisation informatique et le processus que vous allez mettre en place !",
+          "tags": [ "Agile", "Architecture", "Automatisation", "Build", "Continuous Delivery", "DevOps", "Industrialisation", "Intégration", "IT automation", "Open Source", "Ops", "Outils", "Performance", "Scalabilité", "Sécurité", "Stratégie", "supervision", "TDD", "Test", "Tests unitaires", "Usine logicielle" ],
+          "lang": [ "fr", "en", "es" ]
+        }
+      ],
+      "cities": [
+        "Bayonne", "Biarritz", "Saint-Jean-de-Luz", "Bordeaux", "Paris", "San Sebastian"
```

Comment:
I hope we will be able soon to support something like from this city in a area of x km.
But for now it's fine.